src/PieChart/index.js

```diff
@@ -74,10 +75,9 @@ function PieChart({
     legend ||
     (data1 && data1[0].name && data1) ||
     (data2 && data2[0].name && data2);
-
   const legendProps = legendData && {
     colorScale: colorScale1,
-    data: legendData,
+    data: legendData.map(d => ({ name: d.name, label: { text: d.label } })),
```

Comment:
I think this was a mistake in `v0.1`, legend label and data label _need_ not be the same ...e.g. in comparison mode,  there a two data points with the same name but different values and hence labels. With this approach, legend will always show label from `data1` which is incorrect.

We should either not do the automatic setting of legend labels or _combine_ labels from both `data1` & `data2`

Reply:
@kilemensi seems like issue to keep exploring I'll revert the changing so the warning exists. Lets add an issue to track it.